framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -1018,14 +1018,18 @@ public TransferResult<V, S> visitMethodInvocation(
         // add new information based on postcondition
         processPostconditions(n, store, method, invocationTree);
 
-        S thenStore = store;
-        S elseStore = thenStore.copy();
+        if (TypesUtils.isBooleanType(method.getReturnType())) {
```

Comment:
You need to add an `import org.checkerframework.javacutil.TypesUtils` to avoid the compilation error.
Also make sure to reformat the code and run the nullness tests, to have a basic sanity check.